charts/cluster-api-runtime-extensions-nutanix/templates/mindthegap.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Nutanix. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+# This file contains the manifests to run a mindthegap deployment which contains helm charts for our addons.
+# The pod is built via goreleaser with configuration from hack/addons.
+#
+{{ if .Values.selfHostedRegistry }}
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: mindthegap-selfsigned
+  namespace: {{ .Release.Namespace }}
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: mindthegap-tls
+  namespace: {{ .Release.Namespace }}
+spec:
+  dnsNames:
+  - mindthegap.{{ .Release.Namespace }}.svc
+  - mindthegap.{{ .Release.Namespace }}.svc.cluster.local
+  issuerRef:
+    name: mindthegap-selfsigned
+    kind: Issuer
+  secretName: mindthegap-tls
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mindthegap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: mindthegap
```

Comment:
what CAREN clients have to do to move the chart-repository resources to the management cluster? should we apply `clusterctl.cluster.x-k8s.io/move` label ?

Reply:
im not sure any more this would get published and get installed by clusterctl so i dont think this is needed.